csharp/device/Microsoft.Azure.Devices.Client/DeviceAuthenticationWithRegistrySymmetricKey.cs

```diff
@@ -4,11 +4,17 @@
 namespace Microsoft.Azure.Devices.Client
 {
     using System;
+    using Microsoft.Azure.Devices.Client.Extensions;
 
+#if WINDOWS_UWP
```

Comment:
I'm going to revert these (the DeviceAuthentication* classes can be public even with UWP)